setup.py

```diff
@@ -501,7 +501,11 @@ def pxd(name):
         'depends': _pxi_dep['index'],
         'sources': np_datetime_sources},
     '_libs.indexing': {
-        'pyxfile': '_libs/indexing'},
+        'pyxfile': '_libs/indexing',
```

Comment:
why are you doing this?

Reply:
1) Because setup is not at the moment consistent about pxdfiles vs depends.  Being explicit is a step towards fixing that.

2) Because below each `data` in ext_data.values() will define `include = data.get('include', common_include)`, and in cases where `common_include` is not needed, specifying `'include': []` will prevent tossing in unneeded paths.

Reply:
> Because setup is not at the moment consistent about pxdfiles vs depends. Being explicit is a step towards fixing that.

how does this help?

Reply:
I'm not interested in going back down this rabbit hole today.  I'm of the opinion that explicitly listing all of the fields, including empty ones, is going to make this clearer than having different subsets of fields for different entries.  But do what you will.

Reply:
ok, then closing this one. let's try just simply listing the appropriate .pxd and not all of them.